src/client/firebase/getTokenWithHeaders.js

```diff
@@ -4,7 +4,7 @@ export const getTokenWithHeaders = async () => {
   const token = await Firebase.getAuth().currentUser.getIdToken();
   const headers = {
     'Content-Type': 'application/json',
-    authorization: `Sowmya ${token}`,
+    authorization: `${token}`,
```

Comment:
Usually, the convention would be to have `Bearer ` in front of the token but doesn't seem like we are doing that so this looks fine 👍

Reply:
Yes, I have an error on backend with array-destructuring in eslint , so I need to change it here.Otherwise, I would used the text 'Bearer'.

Reply:
@bytewiz Shouldn't we use Bearer in front of token as that is by convention?